### PR TITLE
features/#361-containerized-keyword-support

### PR DESCRIPTION
**Pull request by KrGolovin**
Also adding some tests

### CODE REVIEW

**Comment by KrGolovin** [comment]
containerized

File: src/main/kotlin/com/jetbrains/snakecharm/lang/SnakemakeNames.kt [hunk 1]
```diff
@@ -53,6 +54,7 @@ object SnakemakeNames {
     const val SECTION_RUN = "run"
     const val SECTION_CACHE = "cache" // >= 5.12.0
     const val SECTION_CONTAINER = "container"
+    const val SECTION_CONTAINERIZED = "containerized"
```
Comment (iromeo) [inline src/main/kotlin/com/jetbrains/snakecharm/lang/SnakemakeNames.kt:57]: could you please add a comment in which version the feature was introduced, like for SECTION_CACHER, etc sections here.

File: src/main/kotlin/com/jetbrains/snakecharm/codeInsight/SnakemakeAPI.kt [hunk 2]
```diff
@@ -152,7 +155,8 @@ object SnakemakeAPI {
             SECTION_WRAPPER,
             SECTION_VERSION, SECTION_THREADS,
             SECTION_PRIORITY, SECTION_SINGULARITY, SECTION_CACHE,
-            SECTION_CONTAINER, SECTION_NOTEBOOK, SECTION_ENVMODULES
+            SECTION_CONTAINER, SECTION_CONTAINERIZED, SECTION_NOTEBOOK,
```
Comment (iromeo) [inline src/main/kotlin/com/jetbrains/snakecharm/codeInsight/SnakemakeAPI.kt:158]: Have you checked it on runtime? Is it true, or just an assumption?

Comment (KrGolovin) [reply src/main/kotlin/com/jetbrains/snakecharm/codeInsight/SnakemakeAPI.kt:158]: judging by the documentation, containerized has absolutely the same behavior as the container

Comment (iromeo) [reply src/main/kotlin/com/jetbrains/snakecharm/codeInsight/SnakemakeAPI.kt:158]: Thx, I've also checked the runtime behaviour and agree with you.

File: src/main/kotlin/com/jetbrains/snakecharm/lang/SnakemakeNames.kt (hunk 1 above)
Comment (KrGolovin) [reply src/main/kotlin/com/jetbrains/snakecharm/lang/SnakemakeNames.kt]: https://github.com/JetBrains-Research/snakecharm/pull/381/commits/f0cbee9bad59cc1aa09c2d78a052ecb2880ba48f

**Pull request merged**